dtfs-central-api/src/services/repositories/banks-repo.ts

```diff
@@ -0,0 +1,14 @@
+import db from '../../drivers/db-client';
+import { DB_COLLECTIONS } from '../../constants';
+import { Bank } from '../../types/db-models/banks';
+
+export const getAllBanks = async (): Promise<Bank[] | null> => {
```

Comment:
I think you are correct. `find()` returns some `FindCursor` thingy which I think will just give an empty array from `toArray()` if it can't find anything